packages/block-library/src/site-logo/edit.js

```diff
@@ -564,6 +564,7 @@ export default function LogoEdit( {
 					iconId={ siteIconId }
 					canUserEdit={ canUserEdit }
 				/>
+				<DropZone onFilesDrop={ onFilesDrop } />
```

Comment:
I'm wondering if this should be wrapped in a `canUserEdit` check, as I see another use of `<DropZone>` on [line 640](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/site-logo/edit.js#L640) which is behind a similar check.

Something like this:

```suggestion
				{ canUserEdit && <DropZone onFilesDrop={ onFilesDrop } /> }
```